express-notes-api/server.js

```diff
@@ -0,0 +1,155 @@
+import express from 'express';
+import { readFile, writeFile } from 'node:fs/promises';
+const app = express();
+app.use(express.json());
+
+// errors array
+const errorArr = [
+  { error: 'Id must be a positive integer.' },
+  { error: 'Cannot find note with id.' },
+  { error: 'Content cannot be empty.' },
+  { error: 'An unexpected error occurred.' },
+];
+
+async function parse() {
```

Comment:
I would find this more readable with a name that describes _what_ it is doing instead of _how_. That is, `readData` or similar.